ui/src/message_popup/experience_page.jsx

```diff
@@ -128,17 +128,7 @@ export default React.createClass({
     
     return (
       <div>
-        <AppBar 
-          title='Message PopUp Practice'
-          iconElementLeft=
-          {
-            <IconButton
-              onTouchTap={this.resetExperience}
-              >
-              {!hasStarted ? <MenuIcon/> : questionsAnswered >= sessionLength ? <RefreshIcon /> : <ArrowBackIcon />}
-            </IconButton>
-          }
-          />
```

Comment:
@kevinrobinson Maybe we can add an experience specific section to the Drawer and in there have the options to go back/restart the message popup game?